MyConfig.h

```diff
@@ -1111,6 +1111,12 @@
 #define MY_TRANSPORT_DISCOVERY_INTERVAL_MS (20*60*1000ul)
 #endif
 
+/**
```

Comment:
Could you add a small note on when this feature can/should be used? Maybe some sort of use case description?

Reply:
Hi @mfalkvidd, my use case is a node that is located in a dead zone and can only be reached via a repeater.
When the node sends telegrams to other nodes or the gateway, I always get the error message !TSF:RTE:N2N FAIL in the log. This is a waste of node battery energy and in addition it pollutes the airwaves with traffic that interferes with the communication of other nodes.

Reply:
Do you know what causes the node to try to sent to other nodes?

Reply:
Hi @mfalkvidd, I am intentionally sending the message to another actuator node which unfortunately is not directly reachable via N2N.